plugins/modules/sonic_ntp.py

```diff
@@ -67,6 +67,7 @@
         elements: dict
         description:
           - List of NTP servers.
+          - minpoll and maxpoll are required to be configured together.
```

Comment:
When the requested line is added to the module documentation section, the following sanity error occurs:

```
Running sanity test "validate-modules"
ERROR: Found 1 validate-modules issue(s) which need to be resolved:
ERROR: plugins/modules/sonic_ntp.py:0:0: invalid-documentation: DOCUMENTATION.options.config.suboptions.servers.required_together: extra keys not allowed @ data['options']['config']['suboptions']['servers']['required_together']. Got [['minpoll', 'maxpoll']]
```

Since other collections do have similar lines in their resource module "module" files, this may be a bug introduced into the Ansible validity check after the previous instances were merged. I will send an inquiry to the Ansible Core team about this.

But for this PR, I will drop this request because there is no functional impact and the module documentation added for the PR already clearly states that these fields are required to be specified together.

I will mark this issue "Resolved".